mapper.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+#Written for python3
+import sys
+
+# input comes from STDIN (standard input)
+for line in sys.stdin:
+        list = line.split()
+        #We'll need the 9th column, that's where the DNA data is.
+        gcount=list[9].count('G')

```

Comment:
Maybe it is a waste of space if we make the whole file into text (from CRAM-format) when we only need the 9:th column?